cider/exec/nextgen/jitlib/llvmjit/LLVMJITControlFlow.h

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2022 Intel Corporation.
+ * Copyright (c) OmniSci, Inc. and its affiliates.
```

Comment:
We need to remove ```OmniSci``` license here.

Reply:
If I remove it, license check will fail.